core/interfaces/i_flyout.js

```diff
@@ -198,4 +198,14 @@ IFlyout.prototype.position;
  */
 IFlyout.prototype.isDragTowardWorkspace;
 
+/**
+ * Does this flyout allow you to create a new instance of the given block?
+ * Used for deciding if a block can be "dragged out of" the flyout.
+ * @param {!BlockSvg} block The block to copy from the flyout.
+ * @return {boolean} True if you can create a new instance of the block, false
+ *    otherwise.
+ * @package
+ */
+IFlyout.prototype.isBlockCreatable;
```

Comment:
Is removing the underscore on this name a breaking change?

I still don't understand our policy on external developers overriding `@package` properties in subclasses. But it seems like if this is part of the interface of a flyout, then renaming part of that interface is a breaking chnage.

Reply:
Technically a breaking change, so I renamed the PR to `fix!`.

Reply:
And also added a comment on the breaking change to the PR description.